### PR TITLE
updating the faculty at the University of Central Florida in the depa…

### PR DESCRIPTION
**Pull request by leavens**
…rtment of Computer Science

# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

_Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

**Inclusion criteria**

- [ ] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [ ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

**Comment by leavens** [comment]
I probably shouldn't have closed that, oops.

File: csrankings.csv [hunk 1]
```diff
@@ -97,10 +97,11 @@ Abdullah Muzahid,Texas A&M University,http://people.tamu.edu/~abdullah.muzahid/i
 Abdullah Uz Tansel,CUNY,https://www.gc.cuny.edu/Page-Elements/Academics-Research-Centers-Initiatives/Doctoral-Programs/Computer-Science/Faculty-Bios/Abdullah-Uz-Tansel,emeq3i8AAAAJ
 Abdulmohsen Almalawi,King Abdulaziz University,http://balmalowy.kau.edu.sa/CVEn.aspx?Site_ID=0008859&Lng=EN,XxBxuykAAAAJ
 Abdulmotaleb El-Saddik,University of Ottawa,http://www.site.uottawa.ca/~elsaddik,VcOjgngAAAAJ
-Abedelaziz Mohaisen,University of Central Florida,http://cs.ucf.edu/~mohaisen,q387AywAAAAJ
+Aziz Mohaisen,University of Central Florida,http://cs.ucf.edu/~mohaisen,q387AywAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: Needs all aliases (see https://dblp.uni-trier.de/pers/m/Mohaisen:Aziz.html); please update `dblp-aliases.csv` as needed.

File: csrankings.csv [hunk 2]
```diff
@@ -6358,8 +6361,7 @@ Girish Varma,IIIT Hyderabad,https://www.iiit.ac.in/people/faculty/girish.varma,Y
 Gisele L. Pappa,UFMG,http://homepages.dcc.ufmg.br/~glpappa,C_0ZLuYAAAAJ
 Gisele Lobo Pappa,UFMG,http://homepages.dcc.ufmg.br/~glpappa,C_0ZLuYAAAAJ
 Gita Alaghband,University of Colorado - Denver,http://cse.ucdenver.edu/~gita,NOSCHOLARPAGE
-Gita Reese Sukthankar,University of Central Florida,http://www.eecs.ucf.edu/~gitars,NOSCHOLARPAGE
-Gita Sukthankar,University of Central Florida,http://www.eecs.ucf.edu/~gitars,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings.csv:6362]: Needs to be present; same issue as above (alias). https://dblp.uni-trier.de/pers/s/Sukthankar:Gita_Reese.html

File: csrankings.csv [hunk 3]
```diff
@@ -9208,8 +9209,8 @@ Joseph Hummel,University of Illinois at Chicago,https://www.cs.uic.edu/~jhummel,
 Joseph Izraelevitz,University of Colorado Boulder,https://www.colorado.edu/faculty/izraelevitz,CtQUnFUAAAAJ
 Joseph J. Boutros,Texas A&M at Qatar,http://www.josephboutros.org,lyjl3nMAAAAJ
 Joseph J. DiStefano III,University of California - Los Angeles,http://www.cs.ucla.edu/joseph-distefano-iii,NOSCHOLARPAGE
-Joseph J. LaViola,University of Central Florida,http://www.eecs.ucf.edu/~jjl,kXZaMu8AAAAJ
-Joseph J. LaViola Jr.,University of Central Florida,http://www.eecs.ucf.edu/~jjl,kXZaMu8AAAAJ
+Joseph T. Kider Jr.,University of Central Florida,https://www.ist.ucf.edu/Contact/StaffDirectory/JosephKider.aspx,uQxgKlwAAAAJ
+Joseph J. LaViola Jr.,University of Central Florida,https://www.cs.ucf.edu/~jjl/,kXZaMu8AAAAJ
```
Comment (emeryberger) [inline csrankings.csv:9213]: And here, aliases needed: https://dblp.uni-trier.de/pers/l/LaViola_Jr=:Joseph_J=.html

File: csrankings.csv [hunk 4]
```diff
@@ -10034,6 +10035,7 @@ Kenneth J. Supowit,Ohio State University,https://cse.osu.edu/people/supowit.1,NO
 Kenneth L. Calvert,University of Kentucky,https://www.cs.uky.edu/~calvert,-6v-CdkAAAAJ
 Kenneth M. Anderson,University of Colorado Boulder,http://www.cs.colorado.edu/~kena,wOO8rEMAAAAJ
 Kenneth M. Hopkinson,Air Force Institute of Technology,https://www.afit.edu/BIOS/bio.cfm?facID=87,NOSCHOLARPAGE
+Kenneth O. Stanley,University of Central Florida,https://www.cs.ucf.edu/~kstanley/,6Q6oO1MAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: My understanding is that he is no longer full-time at UCF but instead at OpenAI.

**Review by emeryberger — CHANGES_REQUESTED** [review]
Please restore missing aliases and address other comments - thanks!

**Comment by leavens** [comment]
Hi Emery,

I made the requested changes and pushed them, but the push seems to have gone to leavens/CSrankings so just to my branch, I think. I’m not very clear on what I need to do next to help you. Should I create a new pull request, or is there some way to update the old one?

Sorry about Ken Stanley, I believe we didn’t hear an official word before I made my last update, but you are right that now he has left us…


        Regards,

        Gary T. Leavens
        437D Harris Center
        Computer Science, University of Central Florida
        4000 Central Florida Blvd., Orlando, FL 32916-2362 USA
        http://www.cs.ucf.edu/~leavens phone: +1-407-823-4758
        Leavens@ucf.edu<mailto:Leavens@ucf.edu>

**Comment by emeryberger** [comment]
Looks good.

**Comment by leavens** [comment]
Hi Emery,

Thanks, and sorry for the trouble. It wasn’t clear to me the invariant that governs the CSV files...

        Regards,

        Gary T. Leavens
        437D Harris Center
        Computer Science, University of Central Florida
        4000 Central Florida Blvd., Orlando, FL 32916-2362 USA
        http://www.cs.ucf.edu/~leavens phone: +1-407-823-4758
        Leavens@ucf.edu


On Jul 9, 2020, at 6:49 PM, Emery Berger <notifications@github.com> wrote:

﻿

Looks good.

**Pull request merged**